settings.gradle

```diff
@@ -2,6 +2,10 @@
 include "code/bevzuk-anton"
 project(":code/bevzuk-anton").name = "bevzuk-anton-Model"
 
+// Kildishev Maksim
+include "code/kildishev-maksim"
+project(":code/kildishev-maksim").name = "kildishev-maksim-Model"
```

Comment:
Not sure if everything is correct here.
The name of the project must match the theme of the laboratory work and be without personal information.

But I think it’s worth clarifying at a lecture.

Reply:
Fixed!